CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to UI-Toolkit
+
+A BIG thank you from everyone using the UI-Toolkit for taking the time to contribute! :clap:
+
+The following is a set of guidelines for contributing to the UI-Toolkit and its components,
+These are just guidelines, not rules, use your best judgment and feel free to
+propose changes to this document in a pull request.
+
+This guide will be constantly updated.
+
+## New Components
+* You can create new components easily using the
+[Component Generator](https://github.com/holidayextras/generator-ui-component)
```

Comment:
How do you mean?
[Component Generator](/holidayextras/generator-ui-component) should always resolve forked or not?